env/includes/settings/tk-maya.yml

```diff
@@ -45,7 +46,7 @@ settings.tk-maya.asset_step:
     tk-multi-breakdown:
       location: "@apps.tk-multi-breakdown.location"
     tk-multi-loader2: "@settings.tk-multi-loader2.maya"
-    tk-multi-publish2: "@settings.tk-multi-publish2.maya.asset_step"
+    tk-multi-publish2: "@settings.tk-multi-publish2.maya.asset_step.unreal"
```

Comment:
is `.unreal` intentional here?

Reply:
yes it is: this is to avoid conflicts with the default2 setting. This is the `asset_step` setting for Maya coming from the Unreal integration.